universal-application-tool-0.0.1/app/services/question/types/MultiOptionQuestionDefinition.java

```diff
@@ -102,14 +103,16 @@ protected MultiOptionQuestionDefinition(
         questionHelpText,
         MultiOptionValidationPredicates.create());
     this.options = checkNotNull(options);
-    this.supportedLocales = getSupportedLocales(options);
+    this.supportedOptionLocales = getSupportedOptionLocales(options);
   }
 
+  @Override
   public ImmutableSet<Locale> getSupportedLocales() {
-    return supportedLocales;
+    ImmutableSet<Locale> questionTextLocales = super.getSupportedLocales();
+    return ImmutableSet.copyOf(Sets.intersection(questionTextLocales, this.supportedOptionLocales));
```

Comment:
praise: Nice detail-oriented thinking on this one!!

Reply:
Thanks Natalie!